server/models/tenant.js

```diff
@@ -81,6 +81,11 @@ const tenantSchema = new mongoose.Schema(
         },
       },
     ],
+    profileImageUrl: {
+        type: String,
+        required: true,
```

Comment:
can/should this field be optional?

i was thinking that the tenants might not have to provide their profile image in order to rent a place at a property mgmt company? 
we could make this optional so that we render their images if there's any, and an avatar if there's none

any thoughts? (i didn't attend Wed's meeting so there might be something you guys have discussed that i'm not aware of)

everything else looks great!

Reply:
oh right, that should be false.

Reply:
marking as resolved.